2019-pydays-vienna/typos-correction-with-lookout.md

```diff
@@ -0,0 +1,36 @@
+# Title of the Talk
+
+**Updates**:  https://github.com/src-d/conferences/issues/133
+
+**Sent on**:  -
+
+**Status**:   on preparation
+
+**Author**:   Irina
+
+**Bio**: Junior Machine Learning Engineer at source{d}. Persue Master's degree of Computer Science and Data Analysis at MIPT (Moscow
+Institute of Physics and Technology). Study at Yandex School of Data Analysis.
+
+**Slides**:   -
+
+**Abstract**:
+
+We all make typos in code. Some are filtered at an early stage with help of IDEs or during compilation and tests.
+Still big part of them leaks into the production code: there are more than 1 million typos-connected issues on GitHub (over 100k are open).
+So in the end huge amount of typos get to be ponted-out by human reviewers in PRs (or even in production).
+It takes a lot of time, concentration and manual correction. And it's very annoying. The time has come to bring some automation in.
+
+I present typos lookout analyzer - the tool for automatic correction of typos inside source code in Pool Requests on GitHub.
```

Comment:
The flow should be:

- Motivation, solved problem statement (1-2 sentences)
- What is Lookout (1 sentence)
- Subject: typos corrector. Better to avoid "analyzer" word as it adds cognitive load. Do write why it is cool. (2-3 sentences)
- Results: 95% accuracy /whatever cool number/ (1 sentence)
- What will you talk about: about Lookout, about typos corrector, about your Python+ML development experience in a real-world problem. (1 sentence)

Avoid: any links, any low level details.